controller/array_action/array_mediator_ds8k.py

```diff
@@ -50,6 +54,19 @@ def get_volume_id_from_scsi_identifier(scsi_id):
     return scsi_id[-4:]
 
 
+def get_source_volume_id_if_exists(api_volume):
+    fc_source = [flashcopy.sourcevolume for flashcopy in api_volume.flashcopy if
```

Comment:
`fc_` might be confused with fibre channel. rename to `flashcopy_` or something (wherever applicable)

Reply:
`fcrel`+`fc_rel`+`flashcopy/ies` are better - so it's done